### ISSUE TITLE
Uses for Rare Earth Elements (Whatsapp_2)

### ISSUE THREAD

**Issue by Regian24** [GregTechCEu/gt-ideas#25]
Here's just some uses of some REEs that I know of, not that many, and some are repetitive ._.

Praseodymium Uses - Is used in alloys with nickel to make temperatures down to 1K - 0.001 K, welding goggles because it filters light, it's also uses in silicates to slow down light
To reach those temperatures you have to make coils of an alloy of praseodymium and nickel, you then use the magnetocaloric effect to cool down any object around 1K to 0.001K (which is as far as you can go with the magnetocaloric effect for now using the alloy)

Promethium Uses - Nuclear Photoelectric Battery
Samarium Uses - Neutron absorber, alloy with Cobalt to make magnets, Samarium lexidronam used in cancer treatment, Samarium Sulfide good for conversion of ∆T to electrical energy (like Peltier element) 
Europium Uses - Television screens, good at phosphorescence (it gives red light) 
Gadolinium Uses - Contrast reagent in MRI machines
Terbium Uses - Television screens, good at phosphorescence (it gives yellow and green light), used in magnetic devices, it decrease or increase in size when being in a magnetic field (magnetostrictive property) 
Dysprosium Uses - Neutron absorber, used in production of infrared lasers
Holmium Uses - At low temperatures it can get some very strong magnetic fields, used in YAG dopant for medical uses, infrared lasers for medical use, is as good as boron to absorb neutrons
Erbium Uses - Photographic filter, nuclear poison, amplifier of light in optic fiber, it's used in welding goggles 
Thulium Uses - Infrared lasers for medical use, light bulbs 
Ytterbium Uses - Dopant in ceramic capacitors, as a metal is a conductor but when being pushed under extreme pressure about 14000 atm it becomes a semiconductor 

## Sources
https://ptb.discord.com/channels/701354865217110096/942148434926915655/979435945470349322

**Comment by Spluff5** [comment]
Yes, Dysprosium is actually the strongest neutron absorber of any element.
